__tests__/app.test.js

```diff
@@ -25,11 +36,25 @@ describe("GET /api/topics", () => {
         });
       });
   });
-  it('should respond with a 404: Not Found when passed an endpoint that is incorrect or does not exist', () => {
+});
+
+describe("GET /api", () => {
+  it("should respond with a status code of 200 when successful", () => {
+    return request(app).get("/api").expect(200);
+  });
+  it("should respond with a status code of 200 when successful and an object containing information about the available endpoints", () => {
     return request(app)
-      .get("/api/topocs")
-      .expect(404).then(({body}) => {
-        expect(body.msg).toBe('Not Found')
-      })
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body.endpoints).toBe("object");
+        for (endpoint of Object.keys(body.endpoints)) {
+          expect(body.endpoints[endpoint]).toMatchObject({
+            description: expect.any(String),
+            queries: expect.any(Array),
+            exampleResponse: expect.any(Object),
+          });
```

Comment:
You can just require the endpoints.json file into the test file and use a .toEqual here :)